### PR TITLE
61 eslint matching frontend... well... kind of 😏

### LINKED ISSUE #61: Import eslint rules from web project
## Requirements

- Discuss eslint rules together as a team. 
- Import web project eslint rules if acceptable.

### LINKED ISSUE #85: Fix breaking lint action
The lint action is complaining about imports, yet seems out of sync with the behavior of the linter run on our local machines.

Possibly related to #61 as there is some trouble with the airbnb config: https://stackoverflow.com/questions/59265981/typescript-eslint-missing-file-extension-ts-import-extensions (I tried adding these workarounds in #83 but it's still failing)

### PR DESCRIPTION
**Pull request by ann-kilzer**
Resolves #61 goodbye airbnb and resolves #85

# What changed
Lift over many of the eslint rules from the web project, except: 
1. No magic numbers
2. 🤨 I think there are some opportunities for improvement in the web eslintrc, so we can hash things out in the comments 

# Testing instructions
1. run `yarn lint`
2. 🤞🏻 if the CI/CD lint passes that would be great... #83

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        'block-spacing': ['error', 'always'],
+        'brace-style': ['error', '1tbs', { allowSingleLine: true }],
+        camelcase: 'error',
+        'comma-dangle': ['error', 'never'],
+        'comma-spacing': ['error', { before: false, after: true }],
+        'comma-style': 'error',
+        'computed-property-spacing': ['error', 'never'],
+        'function-paren-newline': ['error', 'consistent'],
+        indent: [
+            'error',
```
Comment (ann-kilzer) [inline .eslintrc.js:75]: I noticed in VSCode that I needed to enable eslint as a formatter or my editor was very unhappy with this

File: prisma/seed.ts [hunk 2]
```diff
@@ -1,103 +1,105 @@
+/* eslint-disable camelcase */
```
Comment (ann-kilzer) [inline prisma/seed.ts:1]: need to find a workaround for that goofy 639_3 variable 🥴

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+            4,
```
Comment (ann-kilzer) [inline .eslintrc.js:76]: Should we just keep it at 2 at this point? @ermish I thought you preferred that and I don't really care

File: prisma/seed.ts (hunk 2 above)
Comment (ann-kilzer) [reply prisma/seed.ts:1]: I love my variable because it's very specific but it also cannot be 🐫

**Review by theyokohamalife — APPROVED** [review]
LGTM

**Pull request merged**